src/ModelManager.ts

```diff
@@ -137,11 +137,15 @@ export class ModelManager {
         if (!this._modelClient) {
             this._modelClient = new ModelClient();
         }
-
+        let domain = "";
+        if (this._modelClient) {
+            domain = this._modelClient.apiHost != null ? this._modelClient.apiHost : "";
+        }
```

Comment:
`const domain = this._modelClient?.apiHost;`

Reply:
`ERROR in ./src/ModelManager.ts 112:41
Module parse failed: Unexpected token (112:41)
File was processed with these loaders:
 * ./node_modules/ts-loader/index.js
You may need an additional loader to handle the result of these loaders.
|             this._modelClient = new ModelClient();
|         }
>         const domain = this._modelClient?.apiHost;
|         if (domain && Utils.isAsync(domain) && Utils.isEditMode()) {
|             Utils.appendClientLibs(domain);
 @ ./src/aem-spa-page-model-manager.ts 13:0-57 13:0-57`

Faced above issue probably due to JS version. Resolving it as this shouldn't be blocking.